### PR TITLE
fix: store email channel role arn to amplify meta output

### PR DESCRIPTION
**Pull request by lazpavel**
#### Description of changes
- stores role arn for email channel notifications to amplify meta channel output section

#### Description of how you validated changes
- manual and unit tests

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-category-notifications/src/channel-email.ts [hunk 1]
```diff
@@ -69,7 +69,10 @@ export const enable = async (context:$TSContext, successMessage: string|undefine
   try {
     const data = await context.exeInfo.pinpointClient.updateEmailChannel(params).promise();
     spinner.succeed(successMessage ?? `The ${channelName} channel has been successfully enabled.`);
-    context.exeInfo.serviceMeta.output[channelName] = data.EmailChannelResponse;
+    context.exeInfo.serviceMeta.output[channelName] = {
+      RoleArn: params.EmailChannelRequest.RoleArn,
```
Comment (lazpavel) [inline packages/amplify-category-notifications/src/channel-email.ts:73]: The only change in the file is setting the RoleArn from the input parameters, this issue is caused by a bug in the pinpoint CDK not returning the RoleArn in the EmailChannelResponse body as stated in this [documentation](https://docs.aws.amazon.com/pinpoint/latest/apireference/apps-application-id-channels-email.html#apps-application-id-channels-email-response-body-emailchannelresponse-example). Filed a ticket against Pinpoint CDK and this change is meant to be a temporary workaround until the root cause is being fixed

File: packages/amplify-category-notifications/src/channel-email.ts [hunk 2]
```diff
@@ -80,17 +83,21 @@ export const enable = async (context:$TSContext, successMessage: string|undefine
...
+      },
+      err,
+    );
   }
 };
 
-const validateInputParams = (channelInput: $TSAny) : $TSAny => {
-  if (!channelInput.FromAddress || !channelInput.Identity || !channelInput.RoleArn) {
+const validateInputParams = (channelInput: $TSAny): $TSAny => {
+  if (!channelInput.FromAddress || !channelInput.Identity) {
```
Comment (lazpavel) [inline packages/amplify-category-notifications/src/channel-email.ts:98]: 2/2 change of the file. RoleArn is an optional field.

**Review by evcodes — APPROVED** [review]

**Review by sobolk — APPROVED** [review]

File: packages/amplify-category-notifications/src/channel-email.ts (hunk 1 above)
Comment (akshbhu) [reply packages/amplify-category-notifications/src/channel-email.ts:73]: Can we do like below if the pinpoint cdk returning undefined or null
```suggestion
      RoleArn: data.EmailChannelResponse ?? params.EmailChannelRequest.RoleArn,
```

Comment (Amplifiyer) [reply packages/amplify-category-notifications/src/channel-email.ts:73]: If the roleArn is an optional field, will it cause problem to send an null or empty string here?

**Pull request merged**